packages/tests/esm/graphql-codegen-joist.cjs

```diff
@@ -0,0 +1,5 @@
+const mappers = { Author: "src/entities#Author", Book: "src/entities#Book" };
+
+const enumValues = {};
+
+module.exports = { mappers, enumValues };
```

Comment:
Copied `grapqhl-codegen-joist.js` to `graphql-codegen-joist.cjs` just to get the prototype running.

When `esm: true` in the config, we should probably produce `grapqhl-codegen-joist.mjs` using ESM syntax.